bears/python/PycodestyleBear.py

```diff
@@ -37,8 +37,10 @@ def create_arguments(
         """
         arguments = [r'--format=%(row)d %(col)d %(code)s %(text)s']
 
+        ignore_values = 'E121, E123, E126, E133, E226, E241, E242, E704, W503'
```

Comment:
Yes :)
The user can see now which values are default and can copy them to his coafile if he wants to keep them.